enatega-multivendor-admin/src/components/Food/Food.jsx

```diff
@@ -399,14 +405,18 @@ function Food(props) {
                   </MenuItem>
                 )}
                 {dataCategories &&
-                  dataCategories.restaurant.categories.map(category => (
-                    <MenuItem
-                      value={category._id}
-                      key={category._id}
-                      style={{ color: 'black' }}>
-                      {category.title}
-                    </MenuItem>
-                  ))}
+                  dataCategories.restaurant.categories
```

Comment:
Use optional chaninng operator dataCategories?.restaurant?.categories

Reply:
also operational chaining is not supported by eslint possibly cause of the ecma script version